.github/workflows/cicd.yml

```diff
@@ -28,40 +28,6 @@ jobs:
       - name: Run TSC
         run: npm run build:ci
 
-  deno:
-    name: Generate Deno compatible code
-    runs-on: ubuntu-latest
-
-    needs: testing
-    # Run workflow only if testing passes
-    if: needs.testing.result == 'success'
-
-    steps:
-      - name: Checkout Project
-        uses: actions/checkout@v2
-
-      - name: Use Node.js v16
-        uses: actions/setup-node@v2
-        with:
-          node-version: 16
-
-      - name: Install Dependencies
-        run: npm ci
-
-      - name: Build Files
-        run: npm run build:deno
-
-      - name: Set Git User and Email
-        run: |
-          git config user.name "github-actions[bot]"
-          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
-
-      - name: Commit and Push New Code
-        run: |
-          git add --all .
-          git commit -m "build: deno build for ${GITHUB_SHA}" || true
-          git push || true
-
```

Comment:
I say keep this, only run on push of non-main branches. As far as I'm aware that should allow auto building deno on forks unless the fork owner has disabled acions, at which point CI will fail on the pull request.

Reply:
It hasn't worked before on several PRs as it'd run on our repository with their origin and the token wouldn't permit access

Reply:
because it was running on the pull_request event which doesn't have access to the fork, when pushing to the fork, the fork has access to push to itself

Reply:
It was running on push and pull_request, and neither worked 🥲

Reply:
Oh now I remember, there's two issues with that:
1. If it somehow lacks access to commit the user has no way of knowing
2. If it does have access to commit, the user needs to push an empty commit so we can run CI on it and merge it

Both of these are quite big issues imo, and a forced requested changes till its fixed vs me or whomever is reviewing having to notice is far more appealing. 

As for your concern about main being outdated, well..worst case it updates with release PRs. It couldn't update main anyways due to branch protections

Reply:
copy copy